webapp/src/components/App/Singlestudy/explore/Debug/Tree/FileTreeItem.tsx

```diff
@@ -32,23 +32,16 @@ function FileTreeItem({ name, content, path }: Props) {
 
   return (
     <TreeItem
-      nodeId={filePath}
+      itemId={filePath}
       label={
-        <Box
-          role="button"
-          sx={{
-            display: "flex",
-            alignItems: "center",
-            ...(isFolderEmpty && { opacity: 0.5 }),
-          }}
-          onClick={handleClick}
-        >
-          <FileIcon sx={{ width: 20, height: "auto", p: 0.2 }} />
-          <span style={{ marginLeft: 4 }}>{name}</span>
+        <Box sx={{ display: "flex" }}>
```

Comment:
you removed empty folder different styles `...(isFolderEmpty && { opacity: 0.5 }),` can you put it back ?